### PR TITLE
fix: add more tests (99% coverage!)

### PR DESCRIPTION
**Pull request by serras**

### CODE REVIEW

**Comment by serras** [comment]
I didn't know what to do with the files in `dist`, so I haven't commited them.

**Comment by serras** [comment]
Ok, according to Coveralls is 97%, but I guess that's good enough xD

File: __tests__/Lens.test.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { alter, ix, lens, mustBePresent } from '../src/Lens'
 import { notFound } from '../src/notFound'
 import { optic, over, preview, set, toArray, view } from '../src/operations'
 
-const friends = ['Alejandro']
+const friends = ['Alejandro', 'Pepe']
```
Comment (kutyel) [inline __tests__/Lens.test.js:6]: Is this @pepegar? 😂

**Review by kutyel — APPROVED** [review]
🔥

File: __tests__/Lens.test.js (hunk 1 above)
Comment (pepegar) [reply __tests__/Lens.test.js:6]: <3

**Pull request merged**